reference/user_defined_functions.rst

```diff
@@ -362,3 +362,31 @@ This example will attempt to bring the shards of the github_events table to the
 ::
 
 	SELECT replicate_table_shards('github_events', max_shard_copies:=10);
+
+master_modify_multiple_shards
```

Comment:
Can we move this to [Table/Shard DDL section](https://citus-doc-preview-pr-91.herokuapp.com/reference/user_defined_functions.html#master-apply-delete-command)? Currently it shows up at the end, and in the 'cluster management and repair functions' section.

The content in that section is no longer just DDL, its also DML, so we might want to update the section heading to reflect that.